Reply/app/src/main/res/values/themes.xml

```diff
@@ -60,20 +60,6 @@
         <item name="shapeAppearanceMediumComponent">@style/ShapeAppearance.Reply.MediumComponent</item>
         <item name="shapeAppearanceLargeComponent">@style/ShapeAppearance.Reply.LargeComponent</item>
 
-        <!--Typography-->
-        <item name="textAppearanceHeadline2">@style/TextAppearance.Reply.DisplayLarge</item>
-        <item name="textAppearanceHeadline3">@style/TextAppearance.Reply.DisplayMedium</item>
-        <item name="textAppearanceHeadline4">@style/TextAppearance.Reply.DisplaySmall</item>
-        <item name="textAppearanceHeadline5">@style/TextAppearance.Reply.HeadlineMedium</item>
-        <item name="textAppearanceHeadline6">@style/TextAppearance.Reply.HeadlineSmall</item>
-        <item name="textAppearanceSubtitle1">@style/TextAppearance.Reply.TitleMedium</item>
-        <item name="textAppearanceSubtitle2">@style/TextAppearance.Reply.TitleSmall</item>
-        <item name="textAppearanceBody1">@style/TextAppearance.Reply.BodyLarge</item>
-        <item name="textAppearanceBody2">@style/TextAppearance.Reply.BodyMedium</item>
-        <item name="textAppearanceButton">@style/TextAppearance.Reply.LabelLarge</item>
-        <item name="textAppearanceCaption">@style/TextAppearance.Reply.BodySmall</item>
-        <item name="textAppearanceOverline">@style/TextAppearance.Reply.LabelSmall</item>
```

Comment:
nice! the remapping throughout the layouts looks good. i'm just wondering why this Typography section is removed from the theme? instead, i think we should be updating the attributes on the left side to M3. e.g.:

        <item name="textAppearanceDisplayLarge">@style/TextAppearance.Reply.DisplayLarge</item>
        <item name="textAppearanceDisplayMedium">@style/TextAppearance.Reply.DisplayMedium</item>
        ...

Reply:
Can you link me to some documentation on why we need this. I thought with Material these will be imported by default.

Reply:
Yup. Will take a look soon.